03-Session-Handling/app.js

```diff
@@ -84,6 +79,7 @@ function runFunction ($rootScope, auth, store, jwtHelper, $location){
   $rootScope.logOut = function () {
     store.remove('profile');
     store.remove('token');
+    auth.signout();
     $location.url('/login');
     $rootScope.authenticated = false;
```

Comment:
@ntotten I am not sure about this decision of using a rootScope variable, perhaps to show how to do session handling on your own, instead of using the SDK provided one (like using `if-user`) directives.